android/app/src/main/java/com/chorestory/fragment/ParentCreateFragment.java

```diff
@@ -4,6 +4,9 @@
 import android.app.TimePickerDialog;
 import android.os.Bundle;
 import android.support.design.widget.FloatingActionButton;
+import android.support.v4.app.FragmentManager;
```

Comment:
let me merge my PR first, and then please migrate these to Android X
https://developer.android.com/jetpack/androidx/migrate